doc/api/util.md

```diff
@@ -125,6 +134,9 @@ util.format(1, 2, 3); // '1 2 3'
 ```
 
 ## util.inherits(constructor, superConstructor)
+<!-- YAML
+added: v0.3.0
```

Comment:
I think of the `added` marks as a reference for the user to understand "how far back can I expect this thing to be in this place", so existence elsewhere before then is kind of irrelevant.